.github/workflows/ci.yaml

```diff
@@ -7,7 +7,7 @@ on:
     types: [run-all-tool-tests-command]
 env:
   GALAXY_FORK: galaxyproject
-  GALAXY_BRANCH: release_24.0
+  GALAXY_BRANCH: release_24.1
```

Comment:
Should we test against 24.2 now?

Reply:
I would say next time :) We do not need to be faster than IUC.